WorkloadManagementSystem/DB/PilotAgentsDB.py

```diff
@@ -196,6 +196,14 @@ def deletePilots( self, pilotIDs, conn = False ):
       return S_ERROR( 'Input argument is not a List' )
 
     failed = []
+
+    from DIRAC.WorkloadManagementSystem.DB.PilotsLoggingDB import PilotsLoggingDB
```

Comment:
You can put this import at the beginning of the file. 

But I think a better way of working here would be to expose PilotAgentsDB().deletePilots() at WMSAdministratorHandler level, and then manage connections to the two DBs (PilotAgentsDB and PilotsLoggingDB) from there: invoking a DB from inside another DB is not the cleanest design. 
Then you would need to make sure that other calls to PilotAgentsDB().deletePilots() are going through the service rather than directly the DB.